VideoWeb/VideoWeb/ClientApp/src/app/app.component.spec.ts

```diff
@@ -24,6 +25,8 @@ describe('AppComponent', () => {
     let titleServiceSpy: jasmine.SpyObj<Title>;
     let pageTrackerServiceSpy: jasmine.SpyObj<PageTrackerService>;
     const mockAdalService = new MockAdalService();
+    const mockEventsService = eventsServiceSpy;
+    let eventsService;
```

Comment:
You probably don't need both variables here

Reply:
removed 👍